source/name_mapper.cpp

```diff
@@ -23,8 +23,8 @@
 #include <unordered_set>
 
 #include "spirv-tools/libspirv.h"
-#include "spirv/1.2/spirv.h"
 
+#include "latest_version_spirv_header.h"
```

Comment:
Please add blank line, so that clang-format doesn't reorder headers.

Reply:
See above